server/src/main/java/game/shooting/matchers/PlayerBoardMatcher.java

```diff
@@ -0,0 +1,46 @@
+package game.shooting.matchers;
+
+import model.preparing.Player;
+import game.shooting.Judge;
+import model.shooting.board.ShootingBoard;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class PlayerBoardMatcher {
```

Comment:
docs also here and everywhere later

Reply:
This issue is tracked in number #38 and will be resolved later on.